### ISSUE TITLE
Version 4.8 TEST time is displayed incorrectly in my timezone

### ISSUE THREAD

**Issue by joelsaacks** [EmuELEC/EmuELEC#1461]
**Describe the bug**
I've got the time set to Africa/Johannesberg in the menu options. The time seems to be shown as UTC
I tried a few others. It shows the same time.

**To Reproduce**
1. Switch on time display
2. Restart
3. Time at the bottom is two hours behind.

**Expected behavior**
Correct time

**Screenshots**

**EmuELEC version:**
 - Version 4.8-TEST

**Device
 - SOC: S095x3
 - Brand: X96
 - Model AIR 4GB/100 Mbit
 - Version:

**Comment by shantigilbert** [comment]
Are you connected to the internet?

**Comment by joelsaacks** [comment]
Yes. 24x7. Time is set as soon as I boot. I'm even scraping at the moment. So definitely so.

**Comment by joelsaacks** [comment]
Time is showing as 18:31 even though its 20:31

**Comment by Langerz82** [comment]
Africa/Johannesberg is GMT+2 so it looks like it's using the GMT and not applying the timezone change.

**Comment by joelsaacks** [comment]
> Africa/Johannesberg is GMT+2 so it looks like it's using the GMT and not applying the timezone change.

Exactly that.
